skglm/tests/test_lbfgs_solver.py

```diff
@@ -26,7 +26,7 @@ def test_lbfgs_L2_logreg(X_sparse):
     # fit L-BFGS
     datafit = compiled_clone(Logistic())
     penalty = compiled_clone(L2(reg))
-    w, *_ = LBFGS(tol=1e-12).solve(X, y, datafit, penalty)
+    w, *_ = LBFGS(tol=1e-5).solve(X, y, datafit, penalty)
```

Comment:
is this a leftover that went through accidentally or is there a reason for the change?